CONTRIBUTING.md

```diff
@@ -24,7 +22,7 @@ This project currently has two branches
 
 ## Testing and Linting
 
-Before any commit make sure to always run `make lint` and `make test` consecutively. It guarantees there's no linting error and all tests pass.
+Run `make test-all`.
```

Comment:
It's important to say that before pushing the PR would be better to run tests and linting to avoid failing checks at Github Actions.

Reply:
I think that's implied? If they care to check the specifics they should read the `Makefile` IMO.